ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -18,22 +18,37 @@
  */
 public class AddFormTest extends TestBase {
     private HomePage homePage;
+    private static String name = "newFormTest";
+    private static String description = "description of new form";
+    private static  String version = "1.2";
     private HeaderPage headerPage;
     private ManageFormsPage manageForm;
     private ClinicianFacingPatientDashboardPage patientDashboardPage;
+    private AdministrationPage administrationPage;
+    private ManageHtmlFormsPage manageHtmlFormsPage;
+    private HtmlFormsPage htmlFormsPage;
 
     @Before
     public void setUp() throws Exception {
         homePage = new HomePage(page);
-        assertPage(homePage);
         headerPage = new HeaderPage(driver);
+        administrationPage = new AdministrationPage(page);
         manageForm = new ManageFormsPage(driver);
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
+
     }
 
-    @Ignore //ignore due to moving forms functionality
     @Test
+    @Category(BuildTests.class)
     public void addFormTest() throws Exception {
+     manageHtmlFormsPage =  homePage.goToAdministration().clickOnManageHtmlForms();
+
+     if (manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '"+ name + "')]")).isEmpty()){
```

Comment:
refactor` manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '"+ name + "')]")` into a boolean method in manageHtmlFormsPage and use it here instead